run_benchmarks.rb

```diff
@@ -275,14 +273,14 @@ def run_benchmarks(ruby:, ruby_description:, categories:, name_filters:, out_pat
 
     # Read the benchmark data
     # Convert times to ms
-    ruby_description, *times = File.readlines(ENV["OUT_CSV_PATH"])
-    times = times.map { |v| 1000 * Float(v) }
-    bench_times[bench_name] = times
+    out_data = JSON.parse(File.read result_json_path)
+    File.unlink(result_json_path)
+    ruby_description = out_data["RUBY_DESCRIPTION"]
 
-    # Read the RSS data, converting bytes to MiB
     if rss
-      bench_rss[bench_name] = File.read(ENV["RSS_CSV_PATH"]).to_i / 1024.0 / 1024.0
```

Comment:
This and the `1000 *` above seem to have broken the output:
```
$ ruby run_benchmarks.rb --rss -e "ruby::ruby" fib
...
itr #44: 206ms
itr #45: 206ms
itr #46: 206ms
itr #47: 207ms
itr #48: 206ms
itr #49: 206ms
RSS: 14.0MiB
Average of last 34, non-warmup iters: 207ms
Total time spent benchmarking: 10s

ruby: ruby 3.1.3p185 (2022-11-24 revision 1a6b16756e) [x86_64-linux]

-----  ---------  ----------  ----------
bench  ruby (ms)  stddev (%)  RSS (MiB) 
fib    0.2        1.2         14675968.0
-----  ---------  ----------  ----------

Output:
./data/output_014.csv
```

I can try to fix it in https://github.com/Shopify/yjit-bench/pull/197